datajunction-query/alembic/env.py

```diff
@@ -28,7 +23,7 @@
 # for 'autogenerate' support
 # from myapp import mymodel
 # target_metadata = mymodel.Base.metadata
-target_metadata = SQLModel.metadata
+target_metadata = None  # pylint: disable=invalid-name
```

Comment:
In this case, shouldn't the target metadata point to the SQLAlchemy base metadata?

Reply:
The interaction between the server and the metadata library now only uses psycopg, so no sqlalchemy engines or models at all meaning there's nothing in sqlalchemy base metadata to use (we still use sqlalchemy to let people configure connections for retrieving data from a catalog). It might seem like a tradeoff since we have to write the migration scripts by hand instead of relying on the sqlalchemy metadata but I actually find the SQL statements in the versioning to be a bit easier to understand and this backend DB is so simple that I don't think it will be an issue.